group.yml

```diff
@@ -36,6 +36,11 @@ arches:
 - s390x
 - aarch64
 
+konflux_arches:
```

Comment:
as pointed out by Yuxiang in https://github.com/openshift-eng/art-tools/pull/1410, we can override arches inside the `konflux` stanza which looks cleaner and easier to prune once we're ready

Reply:
thanks - updated